build.sbt

```diff
@@ -192,17 +199,17 @@ lazy val doc =
       micrositeDocumentationUrl := "usage.html",
       micrositeTheme := "pattern",
       micrositePalette := Map(
-        "brand-primary"     -> "#B58900",
```

Comment:
This is an example of formatting that doesn't look nice because the current formatting is `align.preset=none`.

Reply:
Definitely less aesthetic, but it does have the advantage of shrinking diffs... personally fine with me.